src/util/Conversions.h

```diff
@@ -600,6 +600,6 @@ std::optional<string> convertEntityUriToLangtag(const string& word) {
 // _________________________________________________________
 std::string convertToLanguageTaggedPredicate(const string& pred,
                                              const string& langtag) {
-  return pred + "." + langtag;
+  return '@' + langtag + pred;
```

Comment:
So the parser doesn't accept e.g. "rdfs:label@en"? I think that would be more intuitive